users/rmeli/km_crkbd

```diff
@@ -0,0 +1 @@
+../../keyboards/crkbd/keymaps/rmeli/
```

Comment:
please remove the symlinks

Reply:
Will do. But I added them since I've seen them on other user spaces as well.